optimum/onnx/graph_transformations.py

```diff
@@ -296,3 +168,41 @@ def merge_decoders(
     onnx.checker.check_model(merged_model)
 
     return merged_model
+
+
+def model_to_int32(model: ModelProto) -> ModelProto:
```

Comment:
Maybe name the function to something more explicit:
```suggestion
def slice_arguments_to_int32(model: ModelProto) -> ModelProto:
```
This is only a suggestion but I think that the current name is not clear enough.